Source/WebKit/Shared/RemoteLayerTree/RemoteScrollingCoordinatorTransaction.h

```diff
@@ -54,7 +56,7 @@ class RemoteScrollingCoordinatorTransaction {
     bool clearScrollLatching() const { return m_clearScrollLatching; }
 
 #if !defined(NDEBUG) || !LOG_DISABLED
-    String description() const;
+    WTF::String description() const;
```

Comment:
Hm...it's a tiny bit surprising that we'd need a `WTF::` here. Is that because it conflicts with `API::String` from a nearby unified source?

Reply:
> the compiler "asked" if I meant `std::string` instead

Oh, I see — this is the "hint" part of the error message you'd get if you used `String` without including `WTFString.h` first; including `WTFString.h` above was sufficient to address this.